config/templates/csv-template.yaml

```diff
@@ -36,14 +36,7 @@ spec:
   install:
     strategy: deployment
     spec:
-      clusterPermissions:
-      - serviceAccountName: managed-velero-operator
-        # Rules will be added here by the generate-operator-bundle.py script.
-      permissions:
-      - serviceAccountName: managed-velero-operator
-        # Rules will be added here by the generate-operator-bundle.py script.
-      deployments:
```

Comment:
deployments was removed. Is that intentional?

Reply:
It gets added again [here](https://github.com/openshift/managed-velero-operator/pull/116/files#diff-e12cb709d3181a1ca667b183451c41bbb410dd055344a17d1c0bc981430fde16R288-R294) from the yaml in the deploy dir.

Reply:
Yes, [this](https://github.com/openshift/managed-velero-operator/blob/f6939d65884c5b77b61b3830801614b9428d34c0/boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py#L289-L294) will vivify it correctly.